frontend/src/App.vue

```diff
@@ -126,6 +126,26 @@
             <v-list-item-title>{{ $t("nav.settings") }}</v-list-item-title>
           </v-list-item-content>
         </v-list-item>
+        <v-list-item link :to="{ name: 'Notifications' }">
```

Comment:
* I'd suggest putting the Notifications menu item as second after Dashboard per default
* We should probably remove the partner/wallet badges now

Reply:
Moved the Notifications under Dashboard. I removed the Business Partner badge, but left the Wallet badge because we haven't implemented the manual Credential Offer flow yet.